projects/demo/src/modules/charts/line-days-chart/examples/2/pipe.ts

```diff
@@ -1,21 +1,29 @@
 import {Inject, Pipe, PipeTransform} from '@angular/core';
 import {TuiDay, TuiDayRange, TuiMonth} from '@taiga-ui/cdk';
 import {TUI_MONTHS} from '@taiga-ui/core';
+import {Observable, of} from 'rxjs';
+import {map} from 'rxjs/operators';
 
 // @dynamic
 @Pipe({
-    name: 'labels',
+    name: 'labels$',
```

Comment:
I don't think we should add `$` to pipe's name, it looks confusing as if it's not a pipe but a variable when you see it in template.